internal/kms/repository_audit_key_version.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/hashicorp/boundary/internal/db"
 	"github.com/hashicorp/boundary/internal/errors"
-	wrapping "github.com/hashicorp/go-kms-wrapping"
+	wrapping "github.com/hashicorp/go-kms-wrapping/v2"
```

Comment:
Reviewers: this file could be looked at if desired, very minor changes but repeated throughout the various key types.